### ISSUE TITLE
radio-button: screen reader does not read the labels

### ISSUE THREAD

**Issue by satishsan** [angular/material#11923]
# Demo and steps to reproduce the issue
 Enable Screen reader
1. open a web page with radio button group
2. Navigate to a radio button using keyboard tab key
3. Observe screen reader does not announce the label of the radio button.

**Demo URL (required)***:  

#### Detailed Reproduction Steps
1. 
1. 

### Explain the expected behavior
The Radio buttons are read on first selection of the group 

### Explain the current behavior
The radio buttons are only read on selecting the buttin

### Discuss the use-case or motivation for changing the existing behavior
Accessibility

### List the affected versions of AngularJS, Material, OS, and browsers

- AngularJS: 1.
- AngularJS Material: 1.
- OS: 
- Browsers: 
 
#### Stack Trace
```

```

**Comment by Splaktar** [comment]
@satishsan Thank you for taking the time to submit this issue, your first ever! However it does not follow our **required** issue template. 

- What version of AngularJS Material is this being reported against? 
- Which screen reader and OS can this be observed on?
- Can this be reproduced on the docs page (https://material.angularjs.org/HEAD/demo/radioButton)?

This gives the team the information needed to investigate the bug or prioritize the enhancement.

**Comment by Splaktar** [comment]
You mention

> Observe screen reader does not announce the label of the radio button.

and 

> The Radio buttons are read on first selection of the group

So it sounds like you are reporting two issues?
1. focusing the group does not read all of the radio button labels
1. focusing individual radio buttons does not read the buttons label (until you actually select it)

Is this accurate?

**Comment by satishsan** [comment]
Sorry for the late response that is correct.

**Comment by kittenish** [comment]
We have split this issue into two to be more accurate. 

Please refer to https://github.com/angular/material/issues/11974 and https://github.com/angular/material/issues/11973. Sorry about the confusion, thanks!
